liip/imagine-bundle/1.8/etc/packages/imagine.yaml

```diff
@@ -0,0 +1,36 @@
+#liip_imagine:
+#    
+#    # valid drivers options include "gd" or "gmagick" or "imagick"
```

Comment:
@fabpot it would be great to put the comment on a line containing the error, so that it disappears when being fixed.